.github/workflows/build_aws_sam.yml

```diff
@@ -0,0 +1,27 @@
+name: di-aws-sam
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'aws-sam/**'
+
+  workflow_dispatch:
+    inputs:
+      image_tag:
+        description: "Tag used within docker repository to mark as unique"
+        required: false
+        type: string
+
+jobs:
+  build-and-push:
+    uses: alphagov/di-dockerfiles/.github/workflows/publish_to_ghcr.yml@master
+    with:
+      docker_context: ./aws-sam
+      push_to_ghcr: true
+      ghcr_repo: ghcr.io/alphagov/aws-sam
+      image_tag: ${{ github.event.inputs.image_tag }}
+    secrets:
+      ghcr_username: ${{ secrets.GHCR_USERNAME }}
+      ghcr_token: ${{ secrets.CHCR_TOKEN }}
```

Comment:
```suggestion
```
No longer needed

Reply:
Removed these in the commit below!

Reply:
No you haven't - you can still see these lines in the "files" tab on the PR, and the suggestion is not marked "Outdated".

The commit below changed the *use* of these secrets, but the secrets are still *defined*. You need to remove both *usage* and *definition*.

Reply:
my mistake, I was looking in the other file!